lib/fastlane/plugin/firebase_app_distribution/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module FirebaseAppDistribution
-    VERSION = "0.2.6.pre.1"
+    VERSION = "0.2.7.pre.1"
```

Comment:
Are you planning to upload the version in a separate CL?

Reply:
Yeah I figured I'd start with just a straight merge of this branch.